### DISCUSSION TITLE
How can I change my payment details for copilot.

### DISCUSSION THREAD

**Discussion by PaulLockett** [community/community#28513] [Copilot Conversations]
I initially chose to pay for Copilot annually but I got a company card and they approved the expense on a monthly basis however I don't see a way to change the card and payment plan even after canceling the free trial.

is there a payments support email I should contact?

**Comment by billythekid** [comment]
Yes! Please fill in our [Accounts support form for billing enquiries](https://support.github.com/contact/account?subject=Copilot+billing&type=billing) and someone will get back to you about this. 

Rest assured that someone will be in touch in your original ticket as soon as they can. I'm sure you can imagine a lot of people have Copilot billing questions at the moment as we move towards a paid subscription, so there may be a bit of a delay. We're getting through these as fast as we possibly can. If you do want to send any follow up please do it in the same ticket (by email reply or in your support portal) rather than opening any new tickets. And best of luck for a speedy resolution!

Cheers,
Billy 🤠
